src/generator-enums.cs

```diff
@@ -101,7 +101,7 @@ void GenerateEnum (Type type)
 			print ("");
 			// the *Extensions has the same version requirement as the enum itself
 			PrintPlatformAttributes (type);
-			print ("[CompilerGenerated]");
+			print_generated_code ();
```

Comment:
minor: the naming is unlike the other (e.g. like previous) calls `PrintPlatformAttributes`

Reply:
nm it's an existing API (not sure why since it's a simple one call)